modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+
+=== Tekton Triggers v0.15.0
+
+* This update adds `v1beta1` Go API types. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
++
+With this update, Triggers now supports a new `apiVersion` value, `v1beta1`. The `v1alpha1` version will follow the beta deprecation policy: `v1alpha1` features will be deprecated over a 9-month period.
+
+[id="configuring-node-selection-1-6_{context}"]
+=== Configuring Node Selection
+// TBD - Where does this release note belong?
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR) YAML. The Operator adds these values to all deployments it creates.
++
+To configure node selection for the Operator's controller and webhook deployment, you edit the `Subscription` CR after installing the Operator.
+The `Subscription` CR has the following fields in its specification:
+- `config.nodeSelector`
+- `config.toleration`
++
+To deploy on an infra node, you update the `TektonConfig` and `Subscription` CRs with the infra label as `nodeSelector` and `toleration` so that these labels are applied to the infra node. (link:https://issues.redhat.com/browse/SRVKP-1544[SRVKP-1544])
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+=== CLI 0.21.0 release
+
+* In CLI 0.21.0, support for all `v1alpha1` resources for `clustertask`, `task`, `taskrun`, `pipeline`, and `pipelinerun` commands are deprecated. These resources will be removed after two releases.
+
+=== Tekton Triggers v0.16.0
+
+* In Tekton Triggers v0.16.0, the redundant status tag is removed from the `EventListener` metrics. (link:https://github.com/tektoncd/triggers/pull/1166[#1166])
++
+[IMPORTANT]
+====
+Breaking change: The Status label has been removed from the `eventlistener_http_duration_seconds_*` metric.
+Remove queryies that are based on the status label.
+====
+
+=== Tekton Triggers v0.15.0
+
+* With the current release, the `EventListener` sends a response before Triggers finishes processing. (link:https://github.com/tektoncd/triggers/pull/1132[#1132])
++
+[IMPORTANT]
+====
+Breaking change: With this change, the `EventListener` stops responding with a `201 Created` status code when it creates resources. Instead, it responds with a `202 Accepted` response code.
+====
+
+* The current release removes the `podTemplate` field from the `EventListener`. (link:https://github.com/tektoncd/triggers/pull/1118[#1118])
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the `EventListener` specification. (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
```

Comment:
Same question for this release note - where to place it?

Reply:
This is a deprecation/removal from the Pipeline component of OpenShift Pipelines.

Reply:
Updated. Thanks.